src/solc_0.8/OperatorFilterer/contracts/upgradeable/DefaultOperatorFiltererUpgradeable.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```

Comment:
I think we should align on approach to Pragma version. Do we really need floating pragma here now that we have 2 0.8 sol folders? Auditors recommend not to use a floating version